teamcity/datadog_checks/teamcity/common.py

```diff
@@ -135,6 +135,9 @@ def get_response(check, resource, **kwargs):
             check.log.error("Access denied. Enable guest authentication or check user permissions.")
         check.log.exception("Couldn't fetch resource %s, got code %s", resource_name, resp.status_code)
         raise
+    except requests.exceptions.HTTPError: 
```

Comment:
Duplicate handling of `requests.exceptions.HTTPError`

I think it would be better to fuse this with the first `except`